src/support/command/handle-response.js

```diff
@@ -19,10 +19,9 @@ const checkForErrors = ({ resolveStatus = [] } = {}) => response => {
 
 const filterResponseMessaging = response => {
   if (response.status === 403) {
```

Comment:
@barryadk @conor-ward @ali-smartbear @ponelat @ben-smartbear i've removed this stuff because the backend won't send messages containing these placeholders any more, so this code is effectively redundant.

However, this means that the messages have changed. The messages previously had some extensions similar to those displayed by the client (though without the links) indicating that the client should upgrade.

My question is are we happy to have lost the extensions to these messages or do we need to avoid that regression? If the latter, I can implement a similar resolution strategy here. That would raise a second question - do we want an abstraction for the resolution strategy to avoid duplicating code and tests and is there a convenience shared javascript library I could add that to?

Bear in mind that I have not changed the current behaviour by removing this code, we already have the regression in production because we did not realise that changing the server messages would have this effect. We could also raise another ticket to create the abstraction and fix the regression here.

Thoughts?

Reply:
It would be good to remove it completely. The only reason for the message placeholder replacement is to avoid displaying them to the user. The error messages received from SwaggerHub should be okay to displaying to the user.

Reply:
Sure, it's just that we were previously detecting the placeholders and then displaying a different upgrade message as a a response from the CLI. I don't know whether it's important or not, perhaps not.